packages/suite/src/hooks/firmware/useRebootRequest.ts

```diff
@@ -65,7 +79,7 @@ export const useRebootRequest = (
                     });
             }
         }
-    }, [device, dispatch, phase, method, requestedMode]);
+    }, [connected, mode, dispatch, phase, method, requestedMode, isLocked]);
```

Comment:
quickiest fix is to have something that changes after the first call to connect is issued.